### PR TITLE
data dict release note updates galileo

### PR DESCRIPTION
**Pull request by bullenc**
Adding data dictionary release notes for Galileo release

### CODE REVIEW

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 1]
```diff
@@ -24,6 +25,189 @@
...
+	* New property: `ensat_pathologic_n`
+	* New property: `ensat_pathologic_stage`
+	* New property: `ensat_pathologic_t`
+	* New property: `first_symptom_longest_duration`
+	* New property: `measurement_type`
+	* New property: `melanoma_known_primary`
+	* New property: `tumor_burden`
+	* New property: `ulceration_indicator`
+	* Changes made to `sites_of_involvement`
+		* Changes made to `items`
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:60]: Remove the "items" line and lower the indentation level for the permissible values.  I think this is present in the modified arrays.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 2]
```diff
@@ -24,6 +25,189 @@
...
+		* New permissible value: `Within 3 Months of Surgery`
+* Altered `masked_methylation_array` Entity
+	* Changes made to `platform`
+		* New permissible value: `Illumina Methylation Epic v2`
+* Altered `methylation_beta_value` Entity
+	* Changes made to `platform`
+		* New permissible value: `Illumina Methylation Epic v2`
+* Altered `molecular_test` Entity
+	* New property: `staining_intensity_scale`
+	* Changes made to `gene_symbol`
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:110]: Collapse second_gene_symbol into this one.

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md [hunk 3]
```diff
@@ -24,6 +25,189 @@
...
+* Altered `treatment` Entity
+	* New deprecated property: `treatment_anatomic_site`
+	* New property: `treatment_anatomic_sites`
+	* Changes made to `protocol_identifier`
+		* New permissible value: `P9407`
+
+
+### Bugs Fixed Since Last Release
+
+The [GDC Data Dictionary Viewer](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/) on the [GDC Documentation Site](https://docs.gdc.cancer.gov) correctly displays permissible values for array-type properties.
```
Comment (wwysoc2) [inline docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md:209]: Add an asterisk to this to make it a list.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 1 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Removed 'items' lines 👍

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 2 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Lumped second_gene_symbol changes with gene_symbol changes

File: docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md (hunk 3 above)
Comment (bullenc) [reply docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md]: Asterisked 👍

**Review by wwysoc2 — APPROVED** [review]
lgtm! Let's wait until we get to QA to merge (in case we have some urgent doc work that needs deploying).

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**